solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -548,15 +619,21 @@ contract RandomBeacon is Ownable {
     /// @notice Reports a relay entry timeout.
     function reportRelayEntryTimeout() external {
         uint64 groupId = relay.currentRequest.groupId;
+        uint256 slashingAmount = relay
+            .relayEntrySubmissionFailureSlashingAmount;
         address[] memory groupMembers = sortitionPool.getIDOperators(
             groups.getGroup(groupId).members
         );
 
-        staking.slash(
-            relay.relayEntrySubmissionFailureSlashingAmount,
+        emit RelayEntryTimeoutSlashed(
+            relay.currentRequest.id,
+            slashingAmount,
             groupMembers
         );
 
+        // TODO: Revisit whether the notifier should receive 5% of the slashing amount.
+        staking.seize(slashingAmount, 5, msg.sender, groupMembers);
```

Comment:
In T staking contract, this 5 means 5% of the notification reward (long story short: notifiers are rewarded from a separate pool that is funded from slashed tokens, and they are rewarded right away while slashing happens in a separate TX), see [here](https://github.com/threshold-network/solidity-contracts/blob/fa83b98566b7c786a499fc438d97b4dd3eee0293/contracts/staking/TokenStaking.sol#L1172-L1174).

This percentage should be governable and should be between [0,100].

Reply:
Done in https://github.com/keep-network/keep-core/pull/2707/commits/e0abf511a5bb37282f4f1eb56f13078de1b56e21 as part of #2707